schema/verify/tables/project_revision_002.sql

```diff
@@ -0,0 +1,9 @@
+-- Verify cif:tables/project_revision_002.sql on pg
+
+begin;
+
+select column_name
+from information_schema.columns
+where table_schema='cif' and table_name='project_revision' and column_name='amendment_status';
```

Comment:
This will not fail if the column doesn't exist, since we'll be returning null instead of throwing (see [docs](https://sqitch.org/docs/manual/sqitchtutorial/#trust-but-verify))

We could do something like that (I'm not a fan of the divide by zero trick, even though it's concise):

```suggestion
do $$
begin
  assert (select column_name
  from information_schema.columns
  where table_schema='cif' and table_name='project_revision' and column_name='amendment_status')
  is not null;
end $$;
```

Reply:
I've seen this in a few of the revert files--I've made a tech debt issue to check them out: https://app.zenhub.com/workspaces/climate-action-secretariat-60ca4121764d710011481ca2/issues/bcgov/cas-cif/1049